superduperdb/backends/ibis/utils.py

```diff
@@ -0,0 +1,4 @@
+def get_output_table_name(model_identifier, version):
+    """Get the output table name for the given model."""
+    # use `_` to connect the model_identifier and version
+    return f'_outputs_{model_identifier}_{version}'
```

Comment:
some databases do not support using "/" in table name

Reply:
good spot

Reply:
Can we do `__` (`"_" * 2`)? This may be needed for some kind of name analysis.

Reply:
It’s possible, but it doesn’t seem to make much sense. A more elegant way is to provide a method in the future to handle the table name processing function of a specific database, so we support a lot of databases, and different databases may have different restrictions.

Reply:
@blythed @jieguangzhou 
should we have a helper function in databackend to create table name? 

so for mongodb 
```
def build_table_name(key, model):
    return f'{model}/{key}/{model}'
    
for ibis
def build_table_name:
    return compatible name
```

Reply:
No, I think this feature is just for ibis and sqlalchemy.

Reply:
I think we need a db helper to adapt the different databases in the future.
But we have to change a lot of stuff about query because many methods in the query do not get the db parameter

Reply:
Ok